### PR TITLE
Add libsignal-service implementation with Hyper, Tungstenite and Tokio

### PR DESCRIPTION
**Pull request by gferon**

### CODE REVIEW

File: libsignal-service/Cargo.toml [hunk 1]
```diff
@@ -43,3 +43,5 @@ anyhow = "1.0"
 
 [features]
 prefer-e164 = []
+actix = []
+hyper = []
```
Comment (rubdos) [inline libsignal-service/Cargo.toml:47]: Did we already want to handle this here?

**Review by rubdos — COMMENTED** [review]
I'm not gonna read the whole Hyper thingy, it looks alright ;p

Other than that, we probably want the hyper/actix features in a separate MR?

File: libsignal-service/Cargo.toml (hunk 1 above)
Comment (gferon) [reply libsignal-service/Cargo.toml]: You're right, this is a remnant from... the future. I'll probably open a MR to use features right after this one is merged, but only if it makes sense after I'm done experimenting.

**Comment by gferon** [comment]
@rubdos if it's fine with you, I'll squash and merge this one.

**Pull request merged**